src/App.js

```diff
@@ -14,21 +14,23 @@ import Login from './components/Login/Login';
 import './App.css';
 
 const App = () => (
-  <BrowserRouter>
-    <div className="app-wrapper">
-      <HeaderContainer />
-      <Navbar />
-      <div className="app-wrapper-content">
-        <Route path="/profile/:userId?" render={() => <ProfileContainer />} />
-        <Route path="/dialogs" render={() => <DialogsContainer />} />
-        <Route path="/news" render={() => <News />} />
-        <Route path="/music" render={() => <Music />} />
-        <Route path="/users" render={() => <UsersContainer />} />
-        <Route path="/settings" render={() => <Settings />} />
-        <Route path="/login" render={() => <Login />} />
+  <Suspense fallback="loading">
+    <BrowserRouter>
+      <div className="app-wrapper">
+        <HeaderContainer />
+        <Navbar />
+        <div className="app-wrapper-content">
+          <Route path="/profile/:userId?" render={() => <ProfileContainer />} />
```

Comment:
Consider storing path strings in a separate object, like we did with ROUTES.

Reply:
Yep